homeassistant/components/jellyfin/config_flow.py

```diff
@@ -50,7 +52,7 @@ async def async_step_user(
                 errors["base"] = "unknown"
                 _LOGGER.exception(ex)
             else:
-                await self.async_set_unique_id(userid)
+                await self.async_set_unique_id(client_id)
```

Comment:
Is this really intended as now the integration can be added multiple times for the same account (or maybe that was the case already)?
Would not username be the better option here maybe?
If you need a client_id anyway for the app maybe save one in the entry.data to use that?

Reply:
There is an overall check before this that prevents multiple config entries from being created, so in that regard the id used should not matter. And if in the future multiple configurations would be allowed, using username would prevent configuring two servers with the same username.

As to saving the client_id to entry.data, I'm not really sure I see the advantage of that. Given that the config entry has a specific field to hold a unique id, it made sense to me to use that.

Reply:
Did a look around and could not find anything to block it being added twice for the same username but maybe that is the in the upstream package?
For saving the client id to entry.data that was only if username or something else was used as unique id (as username seems more logical to use as unique_id instead of a random generated id)

Reply:
It's right at the top of the async_step_user function, here: https://github.com/home-assistant/core/blob/17669a728c5817392557436b586147773b445346/homeassistant/components/jellyfin/config_flow.py#L36-L37